### PR TITLE
Liberate triangle primitive from unwrap()

### PR DESCRIPTION
**Pull request by therealprof**
Signed-off-by: Daniel Egger <daniel@eggers-club.de>

Hi! Thank you for helping out with Embedded Graphics development! Please:

- [ ] Check that you've added passing tests and documentation
- [ ] Add a simulator example(s) where applicable
- [x] Add a `CHANGELOG.md` entry in the **Unreleased** section under the appropriate heading (**Added**, **Fixed**, etc)
- [x] Run `rustfmt` on the project
- [x] Run `./build.sh` (Linux/macOS only) and make sure it passes. If you use Windows, check that CI passes once you've opened the PR.

## PR description

Building up an array, iterator, iterating over it with the min/max function and calling unwrap() on the result just to find the minimum or maxmim of three values seemed like overkill so I replace them with two comparisons each instead.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ Embedded Graphics is a `no_std` library for adding graphics features to display
 - Expose `mock_display::MockDisplay` for use in tests, examples, etc in crates that pull embedded-graphics in as a dependency.
 - **(breaking)** #243 All methods on `DrawTarget` that handle drawing to/updating the display are now fallible and return `Result<(), Self::Error>` to allow for error handling in user code. This also affects `Drawable::draw()`, which now propagates any errors that occur while drawing.
 - **(breaking)** #243 `DrawTarget::flush()` is removed. The display driver itself should provide this method if required.
+- Liberate triangles from unwrap()
```
Comment (jamwaffles) [inline CHANGELOG.md]: This can probably go as it's not a public facing change. Other than that this PR is good for merge.

```suggestion
```

Comment (therealprof) [reply CHANGELOG.md]: lol, just following your checklist.

Comment (jamwaffles) [reply CHANGELOG.md]: Yeah I need to change that I think - public changes only from now I think :)

Comment (jamwaffles) [reply CHANGELOG.md]: #253 wahey

Comment (therealprof) [reply CHANGELOG.md]: Fixed. BTW: You may want to disable that merge master into branch misfeature. That only creates horrible git histories with tons of problem potential later.

**Pull request merged**